terragrunt/modules/win-rustup-rs/lambda.tf

```diff
@@ -0,0 +1,13 @@
+module "origin_request" {
+  source = "../aws-lambda"
+
+  providers = {
+    aws = aws.us-east-1
+  }
+
+  name       = "${local.human_readable_name}--origin-request"
+  source_dir = "lambdas/origin-request"
+  handler    = "index.handler"
+  runtime    = "nodejs16.x"
```

Comment:
This runtime is going to be deprecated in a week (https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html), maybe we can switch to a new one?